includes/definitions/discovery/panos.yaml

```diff
@@ -32,9 +32,16 @@ modules:
         count:
             data:
                 -
-                    oid: panSession
-                    value: panSessionActive
-                    num_oid: '.1.3.6.1.4.1.25461.2.1.2.3.{{ $index }}'
+                    oid: panSessionActive
+                    num_oid: '.1.3.6.1.4.1.25461.2.1.2.3.3.{{ $index }}'
                     group: Sessions
                     descr: Active Sessions
-                    high_limit: panSessionMax
+                    high_limit: '.1.3.6.1.4.1.25461.2.1.2.3.2.{{ $index }}'
```

Comment:
Does this work?  I don't think it does...

Reply:
oh yes, it doesn't work, sorry, do you prefer to delete the line or use the precedent line ? (doesn't work too)

Reply:
line deleted

Reply:
you probably need to fix only the num_oid. Everything else should be all right.